checker/checker.go

```diff
@@ -122,7 +122,7 @@ func (c *Checker) Init() (err error) {
 		}
 		instance.sourceDB, err = dbutil.OpenDB(*instance.sourceDBinfo)
 		if err != nil {
-			return errors.Trace(err)
+			return errors.Annotatef(err, "failed to open source DSN %s:***@%s:%d", instance.cfg.From.User, instance.cfg.From.Host, instance.cfg.From.Port)
```

Comment:
shall we differ use and no password scenario, it is more explicit. (although we can detect whether password is used in DSN from the `err`)
if password is not empty, annotate error with `user:***@host:port` ,
without password, annotate error with `user@host:port`

Reply:
I think no need

Reply:
ok